packages/playground/website/src/components/error-report-modal/index.tsx

```diff
@@ -0,0 +1,169 @@
+import { useEffect, useState } from 'react';
+import Modal from '../modal';
+import { addFatalErrorListener, logger } from '@php-wasm/logger';
+import { Button, TextareaControl, TextControl } from '@wordpress/components';
+
+import css from './style.module.css';
+
+export function ErrorReportModal() {
+	const [showModal, setShowModal] = useState(false);
+	const [loading, setLoading] = useState(false);
+	const [text, setText] = useState('');
+	const [logs, setLogs] = useState('');
+	const [url, setUrl] = useState('');
+	const [submitted, setSubmitted] = useState(false);
+	const [submitError, setSubmitError] = useState('');
+
+	useEffect(() => {
+		addFatalErrorListener(logger, (e) => {
+			const error = e as CustomEvent;
+			if (error.detail?.source === 'php-wasm') {
+				setShowModal(true);
+				setText('');
+				setLogs(error.detail.logs.join(''));
+				setUrl(window.location.href);
+			}
+		});
+	}, []);
+
+	function resetForm() {
+		setText('');
+		setLogs('');
+		setUrl('');
+	}
+
+	function resetSubmission() {
+		setSubmitted(false);
+		setSubmitError('');
+	}
+
+	function onClose() {
+		setShowModal(false);
+		resetForm();
+		resetSubmission();
+	}
+
+	async function onSubmit() {
+		setLoading(true);
+		const formdata = new FormData();
+		formdata.append(
+			'message',
+			['What happened?', text, 'Logs', logs, 'Url', url].join('\n\n')
+		);
+		try {
+			const response = await fetch(
+				'https://playground.wordpress.net/logger.php',
+				{
+					method: 'POST',
+					body: formdata,
+				}
+			);
+			setSubmitted(true);
+
+			const body = await response.json();
+			if (!body.ok) {
+				throw new Error(body.error);
+			}
+
+			setSubmitError('');
+			resetForm();
+		} catch (e) {
+			setSubmitError((e as Error).message);
+		} finally {
+			setLoading(false);
+		}
+	}
+
+	function getTitle() {
+		if (!submitted) {
+			return 'Report error';
+		} else if (submitError) {
+			return 'Failed to report the error';
+		} else {
+			return 'Thank you for reporting the error';
+		}
+	}
+
+	function getContent() {
+		if (!submitted) {
+			return `Playground crashed because of an error. You can help
+				resolve the issue by sharing the error details with
+				us.`;
```

Comment:
Nitpick – consistent formatting:

```suggestion
			return (
				<>
					Playground crashed because of an error. You can help
					resolve the issue by sharing the error details with
					us.
				</>
			);
```